lib/tire.rb

```diff
@@ -1,4 +1,5 @@
 require 'rest_client'
+require 'tire/rest_client_cipher_patch'
```

Comment:
`RestClient` has a bug where it is checking for a cipher. RFE has a patch for this bug and that is what is included here.